### PR TITLE
Add support for webauthn in the teleport-cluster helm chart

### PR DESCRIPTION
**Pull request by somethingnew2-0**
Allow for configuring WebAuthN in the teleport-cluster helm chart https://goteleport.com/docs/access-controls/guides/webauthn/

Not sure if this schema for the values.yaml is the most straightforward.  

Also, the `attestationAllowedCas` and `attestationDeniedCas` should probably be mounted as Kubernetes secrets similar to the enterprise `license` or `teleport-tls` certificate/key.  This can be accomplished using the `extraVolumeMounts` value for now if needed.

### CODE REVIEW

**Review by webvictim — COMMENTED** [review]
@somethingnew2-0 Thanks for the PR!

Could you also please add an example values file under https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-cluster/.lint which sets all values you added so that the Helm's linter will check the templated config?

File: examples/chart/teleport-cluster/templates/config.yaml [hunk 1]
```diff
@@ -55,6 +55,23 @@ data:
   {{- end }}
       authentication:
         type: {{ required "authenticationType is required in chart values" .Values.authenticationType }}
+  {{- if .Values.authenticationSecondFactor }}
+    {{- if .Values.authenticationSecondFactor.secondFactor }}
+        second_factor: {{ .Values.authenticationSecondFactor.secondFactor }}
+    {{- end }}
+    {{- if .Values.authenticationSecondFactor.webauthn }}
+        webauthn:
+          rp_id: {{ required "authenticationSecondFactor.webauthn.rpId is required in chart values to enable webauthn" .Values.authenticationSecondFactor.webauthn.rpId }}
```
[the remark this answers is not in the document]
Comment (somethingnew2-0) [reply examples/chart/teleport-cluster/templates/config.yaml]: Makes sense, done.

**Review by webvictim — COMMENTED** [review]
@somethingnew2-0 Could you also please add a values file under https://github.com/gravitational/teleport/tree/master/examples/chart/teleport-cluster/.lint which sets examples for all values you added, so that the Helm's linter will check the templated config renders OK?

After this I can approve/merge/backport the PR. Thanks!

**Comment by somethingnew2-0** [comment]
@webvictim added

**Review by webvictim — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Comment by russjones** [comment]
Will buddy merge in #10777.

**Pull request closed without merging**